dist/twine.js

```diff
@@ -144,6 +146,14 @@ Twine.refreshImmediately = function() {
   }
 };
 
+Twine.register = function(name, constructor) {
```

Comment:
for sure! Initially I figured we might only accept constructors, what about `module` ?